docs/how-to-guides/flush-qos1-events.md

```diff
@@ -0,0 +1,54 @@
+# Flush QoS1 events
+
+QoS1 events are batched and sent as a single batch to optimize the number of network calls made by the SDK. Batching comes with a side effect of event loss if the app is closed during the wait time([maxTimeBetweenTwoBatches]()) for the batch creation.
+SDK also stores all the QoS1 events in a indexedDB database, so last set of events can even be lost if the database is not persisted in next app session.
```

Comment:
@ashishmishra26 "I" should be uppercase in IndexedDB.

Reply:
updated